csrc/scheduler/mma_utils.h

```diff
@@ -65,14 +69,13 @@ void makeTile(TensorView* tv, std::vector<int64_t> tile_sizes);
 //!  defining vectorized loads.
 void orderTiledConcreteIdAsMaybeAllocationDomain(TensorView* tv);
 
-//! Orders the root id ordering of the given tv as
-//! [Device, Batch, Previous Reduction, M, N, K]
-//!  for easier processing of later scheduling steps.
-//!
-//! This matching works on root domain only, and
-//!  will throw if the tv has a loop iterdomain that is
-//!  not a root id.
-void canonicalizeMmaTvOrdering(TensorView* tv);
+//! Orders the leaf ID canonically, and merges dims of the same role
+//! The return value gives the role of each leaf IterDomain in tv.
+std::vector<MatmulDomain> canonicalizeMmaTvOrdering(
```

Comment:
Is the return value of this function used?

Reply:
It's not currently used. I was planning to use this in a future PR to determine the ordering of M and N when we apply this to mma_result. That way we could detect an output transpose and modify the MmaOp inputs to do matmul+transpose efficiently.